### DISCUSSION TITLE
Opening a file on a specific line using cmd+click

### DISCUSSION THREAD

**Discussion by filiperocha** [ghostty-org/ghostty#3892] [Q&A] [answered]
Hi,
coming from iTerm2 and on it, if I cmd+click on a `file:line` it opens the file on my editor at `line`(at least on vscode). Eg:
![2912141441_uIk6Gg](https://github.com/user-attachments/assets/70e2dac6-cfe9-4951-9618-6e27d9bf9d9d)
it opens 
`~/.asdf/installs/ruby/3.4.1/lib/ruby/gems/3.4.0/gems/actionpack-8.0.1/lib/action_controller/metal/basic_implicit_render.rb` on line 8. Is it possible to do this on Ghostty as it can be done for URLs?

Thanks much

**Comment by rhodes-b** [comment, marked as answer]
you are asking for this feature
https://github.com/ghostty-org/ghostty/discussions/2394

**Comment by andrewmartin** [comment]
+1 to this one. Any way we can make it happen? [This looks relevant](https://github.com/ghostty-org/ghostty/issues/1972)
